### ISSUE TITLE
[feature request] option to only allow admin login from specific whitelisted ip's

### ISSUE THREAD

**Issue by kiranos** [owncloud/core#5558]
As a security option, only allow admin login from specific whitelisted ip's which you can specify in the admin settings.

**Comment by PVince81** [comment]
> only owncloud knows the concept of admin

an app could simply call `\OC:$server->getGroupManager()->isAdmin($userId)`.

**Comment by PVince81** [comment]
An app can use the prelogin hook and prevent login if the above check is true and the IP doesn't match the white list.

Would require https://github.com/owncloud/core/issues/13814.

Closing this as I think this is out of scope for core features and can be implemented as app once https://github.com/owncloud/core/issues/13814 is ready.
